team-arrival-checklist/team-arrival-checklist.tex

```diff
@@ -0,0 +1,35 @@
+\documentclass[a4paper, 12pt]{article}
+\usepackage[utf8]{inputenc}
+
+\usepackage[default,osfigures]{opensans}
+
+\usepackage[margin=0.5in]{geometry}
+\usepackage{ltablex}
+\setlength{\parindent}{0cm}
+
+\begin{document}
+
+\begin{center}
+
+    {\huge{Team Arrival Checklist}}
+
+    \medskip
+
+\end{center}
+
+When a team arrives, please check them off on this sheet by filling in their time of arrival.
+\bigskip
+
+{\large{Date}: \hrulefill}
+
+\large
+\def\arraystretch{1.5}
+\begin{tabularx}{\textwidth}{|c|c|c|}
+    \textbf{TLA} & \textbf{Time of Arrival} & \textbf{Special Requirements?} \\
+\hline
+ABC & & \\ \hline
```

Comment:
What's the plan for adding more teams? Just doing it by hand?

Reply:
For now, yes. We can audomate it later